app/models/conversation.rb

```diff
@@ -0,0 +1,9 @@
+class Conversation < ActiveRecord::Base
+  validates :user_id_1,
+  presence: true
+  validates :user_id_2,
+  presence: true
+
+#pass in the id to the controller for the current user and the selected user.
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.